docs/conf.py

```diff
@@ -99,21 +100,29 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = "alabaster"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
 html_theme_options = {
-    'show_related': True
+    "description": "A Python package for reading CCSDS packet data.",
+    "show_related": True,
+    "github_banner": True,
+    "github_button": False,
+    "github_user": "ddasilva",
+    "github_repo": "ccsdspy",
+    "extra_nav_links": {
+        "ccsds blue books": "https://public.ccsds.org/publications/bluebooks.aspx"
```

Comment:
can you style the capitalization "CCSDS Blue Books"?

Reply:
done